package.json

```diff
@@ -52,8 +51,7 @@
     "mocha": "^3.1.2"
   },
   "dependencies": {
-    "babel-runtime": "^6.9.2",
-    "babylon": "^6.8.1",
+    "babylon": "7.0.0-beta.19",
```

Comment:
Transforms are alpha, but the `babylon` itself is beta?
I am assuming this is a test pr to see how new version fits in with the consumers?